src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.BlockStructure.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Structure;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.Options
+{
+    public partial class AutomationObject
+    {
+        public int ShowOutliningForDeclarationLevelConstructs
```

Comment:
>ShowOutliningForDeclarationLevelConstructs [](http://example.com/codeflow?start=19&length=42)

Can this be generated?

Reply:
@tmat Probably, but I don't think it will be in this PR.